proposals/000-ensure-compilation-scalability.md

```diff
@@ -1,56 +1,99 @@
-# SCP-[000]: Improve user experience for builds that use only direct dependencies
+# SCP-[000]: Minimization of compilation classpaths
 
 ## Proposer
 
 Proposed by Stu Hood, Eugene Burmako - Twitter - November 2016
 
 ## Abstract
 
-The most commonly discussed angle of attack for improving the Scala build experience is improving the speed of the compiler. Instead, this document addresses a different angle: improving the experience of using only "direct" dependencies in builds, in order to improve cache hit rates and unblock scaling multiple remote invocations of the compiler.
+The most commonly discussed angle of attack for improving the Scala build experience is speeding up the compiler. Instead, this proposal addresses a different angle: minimizing compilation classpaths to enable distributed builds.
 
-## Background
+## Minimization of compilation classpaths
```

Comment:
Would drop all of these headings down to level 3 (`###`) below a Background/Overview section.